xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1400,6 +1400,12 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
+  if (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK))
+  {
+    m_clockSync.m_enabled = true;
```

Comment:
@mk01 not sure why you think rr would not stabilize. this is playing 23.976 at a 60hz display:

```
12:19:01 T:123145308819456  NOTICE: ----- error: -6.232591, rr: 0.996408, prop: -0.003116, int: 0.000523
12:19:02 T:123145308819456  NOTICE: ----- error: -3.835275, rr: 0.997607, prop: -0.001918, int: 0.000523
12:19:03 T:123145308819456  NOTICE: ----- error: -2.754601, rr: 0.998147, prop: -0.001377, int: 0.000523
12:19:04 T:123145308819456  NOTICE: ----- error: -2.001182, rr: 0.998524, prop: -0.001001, int: 0.000523
12:19:05 T:123145308819456  NOTICE: ----- error: -3.169526, rr: 0.997940, prop: -0.001585, int: 0.000523
12:19:06 T:123145308819456  NOTICE: ----- error: -3.133604, rr: 0.997958, prop: -0.001567, int: 0.000523
12:19:07 T:123145308819456  NOTICE: ----- error: -2.228050, rr: 0.998410, prop: -0.001114, int: 0.000523
12:19:08 T:123145308819456  NOTICE: ----- error: -1.540539, rr: 0.998754, prop: -0.000770, int: 0.000523
12:19:08 T:123145321213952   DEBUG: AddOnLog: VDR VNSI Client: Open - failed to connect to the backend (Operation timed out)
12:19:09 T:123145308819456  NOTICE: ----- error: -2.269183, rr: 0.998390, prop: -0.001135, int: 0.000523
12:19:10 T:123145308819456  NOTICE: ----- error: -3.035096, rr: 0.998007, prop: -0.001518, int: 0.000523
12:19:11 T:123145308819456  NOTICE: ----- error: -3.575782, rr: 0.997737, prop: -0.001788, int: 0.000523
12:19:12 T:123145308819456  NOTICE: ----- error: -2.274717, rr: 0.998387, prop: -0.001137, int: 0.000523
12:19:13 T:123145308819456  NOTICE: ----- error: -1.826530, rr: 0.998611, prop: -0.000913, int: 0.000523
12:19:14 T:123145308819456  NOTICE: ----- error: -2.622464, rr: 0.998213, prop: -0.001311, int: 0.000523
12:19:15 T:123145308819456  NOTICE: ----- error: -3.325181, rr: 0.997862, prop: -0.001663, int: 0.000523
12:19:16 T:123145308819456  NOTICE: ----- error: -4.446331, rr: 0.997301, prop: -0.002223, int: 0.000523
```

this is OSX where audio sink is not as accurate as ALSA. error stays in a band of +-1ms.